### PR TITLE
Rename `_add_data_quality_check_expectation` to be `_add_schema_change_expectation`, which is more accurate

### PR DESCRIPTION
**Pull request by Shinnnyshinshin**
* renaming an internal method to `_add_schema_change_expectation`  which will allow more autogenerated Expectations

### CODE REVIEW

File: tests/integration/actions/test_draft_datasource_config_action.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ def test_running_draft_datasource_config_action(
         "system_users",
         "checkpoints",
         "organizations",
+        "organizations_auth0_orgs",
```
Comment (anthonyburdi) [inline tests/integration/actions/test_draft_datasource_config_action.py:69]: Any idea why this was added?

File: tests/integration/actions/test_list_table_names.py [hunk 2]
```diff
@@ -49,6 +49,7 @@ def test_running_list_table_names_action(
         "datasources",
         "expectation_suites",
         "organizations",
+        "organizations_auth0_orgs",
```
Comment (anthonyburdi) [inline tests/integration/actions/test_list_table_names.py:52]: Any idea why this was added?

**Review by anthonyburdi — CHANGES_REQUESTED** [review]
LGTM - just not sure why the test table names were added.

File: tests/integration/actions/test_draft_datasource_config_action.py (hunk 1 above)
Comment (Shinnnyshinshin) [reply tests/integration/actions/test_draft_datasource_config_action.py:69]: sorry I should have added an explanation here: 

1. Lakitu created a auth0 table in mercury as part of their work here: https://github.com/greatexpectationslabs/mercury/pull/4141, which is being now being added to the DB that is used for integration tests for `cloud` and `runner`.
2. Runner has the equivalent change in the corresponding test [here](https://github.com/greatexpectationslabs/gx-runner/blob/3696e3b939b4f194fe887a173347d7e475c29de9/tests/integration/actions/test_draft_datasource_config_action.py#L68)

Comment (anthonyburdi) [reply tests/integration/actions/test_draft_datasource_config_action.py:69]: All good - thanks for explaining here!

**Review by anthonyburdi — APPROVED** [review]

**Pull request merged**